Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 51;
+	objectVersion = 50;
```

Comment:
Can you explain why we're altering the Example/watchOSSample app?

Reply:
The modification of watchOSSample in Example was made by mistake when I searched for reference. I will discard all the changes in this file.